copy/copy.go

```diff
@@ -668,10 +663,15 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, ba
 		}
 		if reused {
 			logrus.Debugf("Skipping blob %s (already present):", srcInfo.Digest)
+			bar := ic.c.createProgressBar(pool, srcInfo, "blob", "completed: already exists")
```

Comment:
Maybe “skipped” instead of “completed”?

Reply:
Nice suggestion. "skipped" is closer to what's actually happening!